test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/BankAccountControllerSpec.scala

```diff
@@ -218,15 +220,17 @@ class BankAccountControllerSpec
 
     "Business Bank Account" should {
 
-      val businessBankAccount = sample[BankAccountDetails].copy(isBusinessAccount = Some(true))
+      val businessBankAccount = sample[BankAccountDetails]
+      val bankAccountType     = BankAccountType.BusinessBankAccount
 
-      "Let users to upload supporting evidence when the Bank Account Validation succeeds with accountNumberWithSortCodeIsValid = Yes and accountExists = Yes" in forAll(
+      "Let users check their bank details when the Bank Account Validation succeeds with accountNumberWithSortCodeIsValid = Yes and accountExists = Yes" in forAll(
```

Comment:
Don't we need new testcases to check the edit functionality for those for cases you have created?
`(Some,Some), (Some,None), (None,Some), (None,None)